segmentation/utils/example_spatial_analysis_script.py

```diff
@@ -0,0 +1,63 @@
+import numpy as np
+import pandas as pd
+import h5py
+from segmentation.utils import spatial_analysis
+from segmentation.utils import visualize
+from segmentation.utils import spatial_analysis_utils
+from skimage import io
+import importlib
+importlib.reload(spatial_analysis)
+importlib.reload(visualize)
+importlib.reload(spatial_analysis_utils)
+
+# This script takes a cell expression matrix, label maps for the fovs to be included in the analysis, and, depending
+# on the analysis, a threshold matrix for channel or cluster spatial analysis. In channel spatial analysis, cells for
+# a specific fov are identified, and cells of particular phenotypes are compared to each other to check for positive,
+# negative, or no enrichment. To do this, a distance matrix is created from the label_maps, cell phenotypes are
+# identified by their labels in the image and then significant interactions between different populations of phenotypes
+# are recorded. Similar analysis is also done for channel spatial enrichment; however, instead of looking at cell
+# phenotypes, markers positive for specific thresholds are identified and specific interactions are then characterized
+# between them.
+
+# Import data
+# This is the cell expression matrix including data for fovs 6 and 7, their cell labels, marker expression,
+# cell phenotypes, and cell phenotype IDs.
+all_data = pd.read_csv("/Users/jaiveersingh/Desktop/tests/example_expression_matrix.csv")
```

Comment:
Let's move this data inside the repository, so that other people will be able to run the script